test/test_yaml_utils.py

```diff
@@ -212,6 +213,7 @@ def yaml_formatting_fixtures(fixture_filename: str) -> Tuple[str, str, str]:
     (
         "fmt-1.yml",
         "fmt-2.yml",
+        "fmt-3.yml",
```

Comment:
I noticed this test case was missing, so I included it (even though it's not needed for the play/task path discovery utils)